geode-management/src/main/java/org/apache/geode/management/runtime/RebalanceResult.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.runtime;
+
+import java.util.Map;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.management.api.JsonSerializable;
+import org.apache.geode.management.operation.RebalanceOperation;
+
+/**
+ * the final results of a {@link RebalanceOperation}
+ */
+@Experimental
+public interface RebalanceResult extends JsonSerializable {
+  /**
+   * @return for each region, the statistics reported by rebalance
+   */
+  Map<String, PerRegionStats> getRebalanceStats();
+
+  interface PerRegionStats extends JsonSerializable {
```

Comment:
I think you should change the name "PerRegionStats". How about "RebalanceRegionResult"?
This would change the method "getRebalanceStats" to "getRebalanceRegionResults".
Stats is an abbreviation. It is also used in geode for live data.
I also don't think we need to narrow this interface to only having stats in it. We may want to include other info in the future.

Reply:
done